src/langsmith/evaluation-quickstart.mdx

```diff
@@ -3,131 +3,278 @@ title: Evaluation quickstart
+
+<Tip>
+If you prefer to watch a video on getting started with tracing, refer to the datasets and evaluations [Video guide](#video-guide).
+</Tip>
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- **A LangSmith account**: Sign up or log in at [smith.langchain.com](https://smith.langchain.com).
+- **A LangSmith API key**: Follow the [Create an API key](/langsmith/create-account-api-key#create-an-api-key) guide.
+- **An OpenAI API key**: Generate this from the [OpenAI dashboard](https://platform.openai.com/account/api-keys).
+
+**Select the UI or SDK filter for instructions:**
+
+<Tabs>
+<Tab title="UI" icon="window">
+
+## 1. Set workspace secrets
+
+<WorkspaceSecret/>
+
+## 2. Create a prompt
+
+LangSmith's [Prompt Playground](/langsmith/observability-concepts#prompt-playground) makes it possible to run evaluations over different prompts, new models, or test different model configurations.
+
+1. In the [LangSmith UI](https://smith.langchain.com), navigate to the **Playground** under **Prompt Engineering**.
+1. Under the **Prompts** panel. Modify the **SYSTEM** prompt to:
+
+    ```
+    Answer the following question accurately:
+    ```
+
+## 3. Create a dataset
+
+1. Click **Set up Evaluation**, which will open a **New Experiment** table at the bottom of the page.
+1. In the **Select or create a new dataset** dropdown, click the **+ New** button to create a new dataset.
+
+    <div style={{ textAlign: 'center' }}>
+    <img
+        className="block dark:hidden"
+        src="/langsmith/images/playground-system-prompt-light.png"
+        alt="Playground with the edited system prompt and new experiment with the dropdown for creating a new dataset."
+    />
+
+    <img
+        className="hidden dark:block"
+        src="/langsmith/images/playground-system-prompt-dark.png"
+        alt="Playground with the edited system prompt and new experiment with the dropdown for creating a new dataset."
+    />
+    </div>
+
+1. Add the following examples to the dataset:
+
+    | Inputs                                                   | Reference Outputs                                 |
+    | -------------------------------------------------------- | ------------------------------------------------- |
+    | question: Which country is Mount Kilimanjaro located in? | output: Mount Kilimanjaro is located in Tanzania. |
+    | question: What is Earth's lowest point?                  | output: Earth's lowest point is The Dead Sea.     |
+
+1. Click **Save** and enter a name to save your newly created dataset.
+
+## 4. Add an evaluator
+
+1. Click **+ Evaluator** and select **Correctness** from the **Pre-built Evaluator** options.
+1. In the **Correctness** panel, click **Save**.
+
+## 5. Run your evaluation
+
+1. Select <Icon icon="circle-play" /> **Start** on the top right to run your evaluation. This will create an experiment that you can view in full by clicking the experiment name.
+
+    <div style={{ textAlign: 'center' }}>
```

Comment:
i don't think the div style is needed anymore. images are centered through the css style sheet now